cmd/stc/stc.go

```diff
@@ -306,20 +306,26 @@ func signTx(net *StellarNet, key string, e *TransactionEnvelope) error {
 	return nil
 }
 
-func editor(args ...string) {
+func editor(path string, line int) {
 	ed, ok := os.LookupEnv("STCEDITOR")
 	if !ok {
 		ed, ok = os.LookupEnv("EDITOR")
 	}
 	if !ok {
 		ed = "vi"
 	}
-	if path, err := exec.LookPath(ed); err == nil {
-		ed = path
+	edArgv := strings.Split(ed, " ")
```

Comment:
Note that simply splitting on spaces is an imperfect parsing of what could be in the EDITOR env, but it is better than what it is currently doing. Go's stdlib doesn't have a shellwords parser and simply splitting on spaces will cover the majority of simple use cases.